README.md

```diff
@@ -63,10 +122,17 @@ Assume variables x, y, and z are numbers.
 **a)**
 Write an expression for the mean (i.e. average) of x, y, and z.
 
+(x+y+z)/3;
+
 **b)**
 Write a series of expressions to adjust each of x, y, and z halfway toward their mean.
 That is, reset the value of each variable to something based on its previous value.
 
+a = (x+y+z)/3;
+x = x*(a/x);
+y = y*(a/y);
+z = z*(a/z);
+
```

Comment:
2b) One general note: I'm a huge fan of descriptive variable names--things like `a` tend to be harder to parse once you've had some time away from your code. Good names can also help you avoid silly mistakes sometimes. For this problem, you didn't have any such problems, but your equations for the 3 variables are still a bit off. With `x = (x * ( a/x ));` for example, your two `x`s will cancel out, leaving you with `x = a;`.